index.html

```diff
@@ -5,12 +5,138 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="http://fonts.cdnfonts.com/css/cocogoose?styles=44178" rel="stylesheet">    
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+        rel="stylesheet">
   <link rel="stylesheet" href="style.css">  
   <title>World Developers Summit 2022</title>
 </head>
 <body>
-  <h1 id="h1">Hello World</h1> 
+  <header>
+    <div id="heading">
+        <div class="headingImg">
+            <img src="img/logos/facebook.png" class='fb' alt="Facebook logo image">
+            <img src="img/logos/twitter.png" class="twt" alt="twitter logo image">
+        </div>
+        <ul class="headingList">
+            <li>English</li>
+            <li>My Page</li>
+            <li>Logout</li>
+        </ul>
+    </div>
+    <nav class="mobileMenu">
+        <a href="index.html">
+            <div class="logo hide">                
+              <img src="img/logos/icon.png" alt="3d Logo">
+              <p class="logoText">Developer <br> Summit 2022</p>               
+            </div>
+        </a>
+      <ul class="navList hide">
+          <li><a href="about.html" class="listAnchor">About</a></li>
+          <li><a href="#2" class="listAnchor">Program</a></li>
+          <li><a href="#3" class="listAnchor">Join</a></li>
+          <li><a href="#4" class="listAnchor">Sponsor</a></li>
+          <li><a href="#5" class="listAnchor">News</a></li>
+          <li><a href="#6" class="listAnchor">3D Campaing</a></li>
+      </ul>
+      <img src="img/buttons/mobile_menu.png" id="btnImg" alt="Hamburger type button to display menu">
+  </nav>
+  <div class='txtContainer'>
+      <h1 class="txtGreeting">"Hello! Developers"</h1>
+      <div>
+          <h2 class='textTitle'>Developers Global Summit 2022</h2>
+      </div>
+      <div class="txtParagraph">
+          <p>You will have the chance to learn the latest software development testing technologies, the best
+              practices, and discover new ways to solve complex development problems.
+          </p>
+      </div>
+      <div class="dateContainer">
+          <h2 class="txtDate">2022.10.19(MON) ~ 20(TUE)</h2>
```

Comment:
- [OPTIONAL] Kindly use another font for numbers. It's better to see numeric information instead of unreadable signs. It's a suggestion.